### PR TITLE
Update run-analyzer-macos-linux.md

### LINKED ISSUE #12698: checksum did not match
https://github.com/MicrosoftDocs/microsoft-365-docs/blame/567adda26f355f54d167525b68e7b87d8cad1cf9/microsoft-365/security/defender-endpoint/run-analyzer-macos-linux.md#L51

Checksum returned by the supplied command does not match.

### PR DESCRIPTION
**Pull request by msbemba**
Made the changes as the Checksum returned by the supplied command does not match.

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12698

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/run-analyzer-macos-linux.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ If using a terminal download using the command:
 2. Verify the download.
 
     > [!NOTE]
-    > The current SHA256 hash of 'XMDEClientAnalyzerBinary.zip' that is downloaded from the above link is: '845C9155DDF039990A28EB1EE56E4ACECA706E02C7B78571B99EDB16AACA2F2F'
+    > The current SHA256 hash of 'XMDEClientAnalyzerBinary.zip' that is downloaded from the above link is: 'E8D1B752A937E9AB305AE3C30737E31D75AE6FF9299002AB23F5C463C77DD159'
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/run-analyzer-macos-linux.md]:
```suggestion
    > The current SHA256 hash of 'XMDEClientAnalyzerBinary.zip' that is downloaded from this link is: 'E8D1B752A937E9AB305AE3C30737E31D75AE6FF9299002AB23F5C463C77DD159'
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder. Thanks

**Pull request merged**